### ISSUE TITLE
JSON Dump error

### ISSUE THREAD

**Issue by tscritch** [hfaran/slack-export-viewer#51]
Using python v2.7
`python app.py -z MySlackExport.zip`

gets this error when extracting the zip:
```Traceback (most recent call last):
  File "app.py", line 6, in <module>
    main()
  File "/Library/Python/2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/tscritchfield/Documents/slack-export-viewer/slackviewer/main.py", line 86, in main
    configure_app(app, archive, debug)
  File "/Users/tscritchfield/Documents/slack-export-viewer/slackviewer/main.py", line 43, in configure_app
    path = extract_archive(archive)
  File "/Users/tscritchfield/Documents/slack-export-viewer/slackviewer/archive.py", line 203, in extract_archive
    json.dump(archive_info, f)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 190, in dump
    fp.write(chunk)
TypeError: must be unicode, not str
```

Is there another json dump argument or json to fix this? 
I'll do some research to resolve this today.

**Comment by tscritch** [comment]
This is fixed using python v3.6. Any suggestions to resolve for v2.7?

**Comment by hfaran** [comment]
Fixed in 0.3.3! This also prompted me to finally set up CI for this project on Travis. Hopefully breaks between py2 and py3 won't slip through in the future with that :D

**Comment by tscritch** [comment]
Looks good! Thanks! Yay for CI!
